scripts/setup-macos.sh

```diff
@@ -32,7 +32,11 @@ FB_OS_VERSION=v2021.05.10.00
 NPROC=$(sysctl -n hw.physicalcpu)
 COMPILER_FLAGS="-mavx2 -mfma -mavx -mf16c -masm=intel -mlzcnt"
 DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)}
-MACOS_DEPS="ninja cmake ccache protobuf icu4c boost gflags glog libevent lz4 lzo snappy xz zstd"
+
+MACOS_DEPS_BREW="ninja cmake ccache protobuf icu4c boost gflags glog libevent lz4 lzo snappy xz zstd"
+MACOS_DEPS_MACPORTS="ninja cmake ccache protobuf3-cpp icu boost gflags google-glog libevent lz4 lzo snappy xz zstd openssl"
```

Comment:
While I welcome support for anything other than `brew`, the `brew` path is tested in the CI and is therefore unlikely to break. The macports path isn't; will you sign up to maintain it?

Reply:
I think you just motivated me to install `brew` :)

Reply:
I too both dislike and use brew. Welcome to the club!